crates/sncast/src/starknet_commands/script.rs

```diff
@@ -49,6 +46,8 @@ use starknet::providers::jsonrpc::HttpTransport;
 use starknet::providers::JsonRpcClient;
 use tokio::runtime::Runtime;
 
+type ScriptStarknetContractArtifacts = StarknetContractArtifacts;
```

Comment:
Why do we need this alias?

Reply:
Generally aliases are not very good as those do not provide any type safety. If you want such you should use `struct ScriptStarknetContractArtifacts(StarknetContractArtifacts)`, though I don't feel this provides any useful information here.

Reply:
I have added it just to not use StarknetContractArtifacts for lib paths injection directly. We need this to make sure script artifacts can be found (and this also removes requirement for users to add [lib] parts in their Scarb.toml)

Reply:
I would drop the alias completely and rename `StarknetContractArtifacts` to `CairoArtifact` or something like this (represent artifacts hierarchy properly). Can be done in a separate PR.